packages/testcases/View Guided Answers Content.md

```diff
@@ -0,0 +1,79 @@
+
+Pre Req: Guided Answers Extension is installed
+
+
+### Open Guided Answers (SBAS and VSCODE)
+
+**GIVEN** the user opens in the 'Command Pallette' in SBAS/VSCODE
+**WHEN** the user enters 'SAP: Open Guided Answers'
+**THEN** 'SAP:Open Guided Answers' appears in the command palette drop down
+
+
+**GIVEN** 'SAP: Open Guided Answes' appears in the command paletted dropdown 
```

Comment:
- Spell Check: It should be `SAP: Open Guided Answers`
- Spell Check: It should be `palette`